messaging/src/test/java/org/springframework/security/messaging/handler/invocation/reactive/CurrentSecurityContextArgumentResolverTests.java

```diff
@@ -186,6 +223,25 @@ private MethodParameter arg0(String methodName) {
 
 	}
 
+	@Target({ ElementType.PARAMETER })
+	@Retention(RetentionPolicy.RUNTIME)
+	@CurrentSecurityContext
+	@interface CurrentAuthentication2 {
+
+		@AliasFor(annotation = CurrentSecurityContext.class)
+		String expression() default "";
+
+	}
+
+	@Target({ ElementType.PARAMETER })
+	@Retention(RetentionPolicy.RUNTIME)
+	@CurrentSecurityContext(expression = "authentication.{property}")
+	@interface CurrentAuthentication3 {
```

Comment:
Will you please give this and the other `CurrentAuthentication3` meaningful names? Maybe `CurrentAuthenticationProperty`.

Reply:
Done